packages/perseus-score/src/widgets/interactive-graph/score-interactive-graph.ts

```diff
@@ -247,10 +248,28 @@ function scoreInteractiveGraph(
             userInput.type === "angle" &&
             rubric.correct.type === "angle"
         ) {
-            const guess = userInput.coords;
+            const coords = userInput.coords;
             const correct = rubric.correct.coords;
             const allowReflexAngles = rubric.correct.allowReflexAngles;
 
+            // While the angle graph should always have 3 points, our types
+            // technically allow for null values. We'll check for that here.
+            if (!coords) {
```

Comment:
There's really no possible way for coords to ever be less than 3 coordinates, due to how the graphs are set up. 

I will make another ticket for setting a stricter type for the Angle Graph Coords, as it will require updating several tests that have only partially mocked data.

Reply:
Could you make the ticket now and update this comment to start with a TODO(LEMS-XXXX)?